tests/unit/components/menu-top/MenuTop.spec.ts

```diff
@@ -11,6 +11,7 @@ describe("components/menu-top/MenuTop", () => {
   let store: Store<CalChartState>;
 
   beforeEach(() => {
+    window.URL.createObjectURL = jest.fn();
```

Comment:
Added this because in CLI unit tests there is no URL.createObjectURL...

Reply:
Yeah, feel free to add any mocks that you need!